test/bodyEncoding.js

```diff
@@ -21,15 +21,17 @@ describe('body encoding', function () {
 
   this.timeout(10000);
 
-  var pngHex = '89504e470d0a1a0a0' +
-               '000000d4948445200' +
-               '00000100000001080' +
-               '60000001f15c48900' +
-               '00000a49444154789' +
-               'c6300010000050001' +
-               '0d0a2db4000000004' +
-               '9454e44ae426082';
-  var pngData = new Buffer(pngHex, 'hex');
+  var pngHex = [
```

Comment:
This got auto-formatted with the alignment broken, so I've changed the concat thing a bit